index.html

```diff
@@ -359,16 +359,16 @@
               </li>
               <li>If <a>obtaining the manifest</a> results in an error, a user
               agent can, at this point, fall back to using the <a>top-level
-              browsing context</a>' <code>Document</code>'s metadata to
-              populate an <a>installation process</a>' UI.
+              browsing context</a> <code>Document</code>'s metadata to populate
+              an <a>installation process</a> UI.
```

Comment:
I think this should read "populate the installation process's UI" (i.e., s/an/the) because the installation process was already instantiated by Step 2.1.

I think the apostrophe here was the author following Rule 1c [here](http://www.grammarbook.com/punctuation/apostro.asp): "Some writers and editors add only an apostrophe to all nouns ending in s." So it is correct by some standards, but I think it's pretty unusual and better with an 's.